rerun-failed-run/rerun-failed-run.yml

```
@@ -0,0 +1,73 @@
+name: Retrigger failed run action
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: The id of the run which we want to retrigger
+        required: true
+      error_message:
+        description: Custom error message to look for in the logs.
```

For consistency with the other parameters' description fields:
```suggestion
        description: Custom error message to look for in the logs
```